src/com/shashi/beans/Discount.java

```diff
@@ -0,0 +1,125 @@
+package com.shashi.beans;
+
+import java.time.LocalDate;
+import java.time.Period;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import com.shashi.utility.DBUtil;
+
+public class Discount {
+    private String productId;
```

Comment:
We don't need productId here as a discount can apply to many products. This should be a many (product) to one (discount) relationship. Each product should have an optional discount column, that way if you need to change a discount, you change it once and it will apply to all linked products